packages/model/addon/-private/model.js

```diff
@@ -1862,9 +1722,36 @@ Model.reopenClass({
    @type Map
    @readOnly
    */
-  relationshipsByName: relationshipsByNameDescriptor,
+  @computed
+  static get relationshipsByName() {
```

Comment:
We need to cache all of the static relationships stuff for performance.

Reply:
Anything different we need to do or just add a comment?

Reply:
We need to cache the map thats being created inside the method, do you know if computed will cache by default or we need to do it manually

Reply:
computed will cache by default

Reply:
but we should move to `@cached` in the next pass